docs/en/latest/getting-started.md

```diff
@@ -115,55 +117,74 @@ We receive a JSON response when we send the request:
 }
 ```
 
-## Pre-Requisites
+## Install Apache APISIX
 
-Before you jump ahead, make sure that you have your machine setup with these tools.
+To get started, we will install Apache APISIX with Docker and enable the [Admin API](./admin-api.md).
 
-- [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/).
+First clone the [apisix-docker](https://github.com/apache/apisix-docker) repository:
 
-- [curl](https://curl.se/docs/manpage.html) for testing the API. Alternatively, you can use tools like [Hoppscotch](https://hoppscotch.io/) or [Postman](https://www.postman.com/).
+```shell
+git clone https://github.com/apache/apisix-docker.git
+cd apisix-docker/example
+```
 
-<!--
-#
-#    In addition to the basic Markdown syntax, we use remark-admonitions
-#    alongside MDX to add support for admonitions. Admonitions are wrapped
-#    by a set of 3 colons.
-#    Please refer to https://docusaurus.io/docs/next/markdown-features/admonitions
-#    for more detail.
-#
--->
+Now, you can use `docker-compose` to start APISIX.
```

Comment:
Installing APISIX using `docker-compose` will install both Grafana and Prometheus and Nginx and APISIX Dashboard, should we tell users?

Reply:
It did not install Grafana and Prometheus. Nginx is expected as it is the APISIX dataplane. I used the M1 docker-compose file.

![Screenshot 2022-04-28 at 10 12 58 AM](https://user-images.githubusercontent.com/49474499/165678128-ec22152d-2a96-4ab1-8e6d-60305fe53d9b.png)

Reply:
In fact, if you are using CentOS 7, Grafana and Prometheus will be started.
![image](https://user-images.githubusercontent.com/97138894/165680354-558b0141-f3ea-41cb-84b5-8d0a1a91dddf.png)

![image](https://user-images.githubusercontent.com/97138894/165680751-175c4a3d-d705-44ab-ae8d-f41f4b2bd3fe.png)

Reply:
So, you installed with RPM?

Reply:
The consensus was that we will update the docker-compose files to match.